lib/seed/Seeder.js

```diff
@@ -98,7 +112,7 @@ class Seeder {
     const seedPath = this._getNewStubFilePath(name);
     await writeJsFileUsingTemplate(
       seedPath,
-      this._getStubPath(),
+      await this._getStubPath(),
```

Comment:
I would prefer not to have inline await, better extract to a separate const

Reply:
I agree. It confuses the order in which things happen a bit.